versioned_docs/version-1.3/plugins/verifier/notation.md

```diff
@@ -260,3 +260,12 @@ Sample Notation CLI config with timestamping configuration:
     }
 }
```

Comment:
You may need to submit this change to this folder https://github.com/ratify-project/ratify-web/tree/main/docs. The versioned folder `v1.4` will be created after the release.

Reply:
Good point